packages/test/test-end-to-end-tests/src/test/localTestSignals.spec.ts

```diff
@@ -191,3 +196,224 @@ describeCompat("TestSignals", "FullCompat", (getTestObjectProvider) => {
 		);
 	});
 });
+
+describeCompat.skip("TargetedSignals", "NoCompat", (getTestObjectProvider) => {
```

Comment:
Relocating comment from #19367:
> Everything is skipped. Is there nothing that can be tested? (Hint we definitely should be able to test current behavior.)

I am not sure why #19367 was closed. There is always value in setting up tests before making functional changes. Perhaps things are unblocked if that targetClientId and feature check were removed from DocumentDeltaConnection - we think check is bogus.

Reply:
Or perhaps those tests are not quite the right ones for existing state of the world.

Reply:
Right, I will re-open and update it!